connect-inject/container_init.go

```diff
@@ -131,7 +141,14 @@ services {
 
     {{ range .Upstreams -}}
     upstreams {
+		{{- if .Name }}
```

Comment:
```suggestion
      {{- if .Name }}
```

Looks like the whitespace beast got us here too!